src/components/PDF/Applications.tsx

```diff
@@ -139,43 +146,41 @@ class Applications extends Component<Props, State, {}> {
     );
   }
 
-  changeCurrentPage = (newCurrentPage: number) => {
-    this.setState({ currentPage: newCurrentPage }, this.getDocuments);
-  }
-
-  onChangeViewPermission = (event: any) => {
-    const {
-      currentUser,
-    } = this.state;
-    currentUser.viewPermission = event.target.ischecked;
-    this.setState({ currentUser }, this.getDocuments);
-  }
-
-  getDocuments = () => {
-    const {
-      itemsPerPageSelected,
-    } = this.state;
-    const itemsPerPage = Number(itemsPerPageSelected.value);
-    // fetch call here to get all the current Documents to fill
-  }
+  // TODO: Pagination
+  // changeCurrentPage = (newCurrentPage: number) => {
+  // this.setState({ currentPage: newCurrentPage }, this.getDocuments);
+  // }
+  // }
+  //
+  // handleChangeItemsPerPage = (itemsPerPageSelected: any) => {
+  //  this.setState({
+  //    currentPage: 0,
+  //  });
+  // }
+  // getDocuments = () => {
+  //   const {
+  //     itemsPerPageSelected,
+  //   } = this.state;
+  //   const itemsPerPage = Number(itemsPerPageSelected.value);
+  //   // fetch call here to get all the current Documents to fill
+  // }
 
   render() {
     const {
       currentApplicationFilename,
       currentApplicationId,
-      currentPage,
-      itemsPerPageSelected,
       numElements,
       documents,
     } = this.state;
 
-    const itemsPerPage = Number(itemsPerPageSelected.value);
-    const tablePageSelector = TablePageSelector({
-      currentPage,
-      itemsPerPage,
-      numElements,
-      changeCurrentPage: this.changeCurrentPage,
-    });
+    // TODO: Pagination
+    // const itemsPerPage = Number(itemsPerPageSelected.value);
+    // const tablePageSelector = TablePageSelector({
+    //   currentPage,
+    //   itemsPerPage,
+    //   numElements,
+    //   changeCurrentPage: this.changeCurrentPage,
+    // });
```

Comment:
can you explain what pagination will involve? why dont we have pagination implementation in this PR?

Reply:
Basically, the table pagination doesn't work

Reply:
It would need to work for the future though. Right now, there is no pagination, and it just unwraps into a long list

Reply:
Aren't you doing the table refactor?

Reply:
Is there a table in this component? Or is the pagination just for different pages of the application? The table refactor is specifically for tables and includes the table and pagination as one component.